fixes #1672

sadnixon opened the pull request:
These changes make it so that whenever a:
• vote
• chancellor pick
•inv/SE/shot/peek
• veto
is forced by timer, the game chat gets a little message that says "The timer has forced [playername] {i} to pick/vote/peek."

I didn't add notifications for the actions of president picking discard or chancellor picking policy because those two actions DO get a notification right now, but only to mod chat, and I thought it might be good to leave it that way.

nch0w commented on routes/socket/game/common.js:
```diff
@@ -187,7 +187,24 @@ module.exports.startElection = (game, specialElectionPresidentIndex) => {
+							},
+							{
+								text: game.general.blindMode
+									? `${replacementNames[presidentIndex]} {${presidentIndex + 1}} `
```
`replacementNames` is not defined here, please use `game.general.replacementNames`!

And also test blind games 😉

sadnixon replied:
thanks, believe i have solved this

nch0w reviewed (changes requested):
This does not work in blind games. 🥺

nch0w commented on routes/socket/game/election.js:
```diff
@@ -260,6 +260,27 @@ const enactPolicy = (game, team, socket) => {
+											type: 'player'
+										},
+										{
+											text: powerToEnact[1]==='The president must examine the top 3 policies.'
```
Lint please (there should be spaces around "===")

sadnixon replied:
there we go

Vigasaurus commented:
I personally think this needs a mod consensus - I can see it both ways that a. You want to always show everyone if an action was a timer action, or b. The former may cause some amount of intentional suboptimal by people using the timer to get out of making "non-gamethrowy" plays, and having that be confirmed by the game may be a problem. 

I think it ought to be one or the other, either **all** actions' "timer-ness" is sent to modchat only, or all of them (including elections) are sent to general chat only.

sadnixon commented:
Went ahead and added in timer notifs on everything for consistency's sake. Didn't remove the mod messages, just added the new public ones (since the mod ones give additional info that we still want).

Bruno623 commented:
This was discussed amongst the Moderators and we've decided against its implementation
